manifests/2.4.0/opensearch-2.4.0.yml

```diff
@@ -25,3 +25,11 @@ components:
   - name: security
     repository: https://github.com/opensearch-project/security.git
     ref: 2.x
+  - name: performance-analyzer
+    repository: https://github.com/opensearch-project/performance-analyzer.git
+    ref: main
```

Comment:
Hi @sruti1312 , is PA not following the standard branching strategy yet?
Main pointing to `3.0` and `2.x` tracking next minor, in this case `2.4`?

Reply:
Hi @gaiksaya. I have created the 2.x branch to track the next minor release and have opened PR's to update the main to 3.0. Thanks!

Reply:
Thanks! I think you will also need to update https://github.com/opensearch-project/performance-analyzer/issues/241 since this is related to your effort